S/SDPA/build_tarballs.jl

```diff
@@ -94,29 +98,27 @@ products = [
     LibraryProduct("libsdpawrap", :libsdpawrap)
 ]
 
-# Pick platforms from L/libcxxwrap-julia/build_tarballs.jl
-platforms = [
-    FreeBSD(:x86_64; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    Linux(:armv7l; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    Linux(:aarch64; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    Linux(:x86_64; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    Linux(:i686; libc=:glibc, compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    MacOS(:x86_64; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    Windows(:x86_64; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-    Windows(:i686; compiler_abi=CompilerABI(libgfortran_version=v"4", cxxstring_abi=:cxx11)),
-]
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+include("../../L/libjulia/common.jl")
+platforms = libjulia_platforms(julia_version)
+platforms = expand_cxxstring_abis(platforms)
 #platforms = expand_gfortran_versions(platforms)
+for p in platforms
+    p["libgfortran_version"] = string(v"4")
+end
```

Comment:
The above is an attempt to imitate what's been in the old platforms list; but I wonder why this JLL recipe doesn't simply use `expand_gfortran_versions`? IMHO it should either use `expand_gfortran_versions`, or else a comment should be added which explains why it is not used.

Reply:
IIRC, we did that because of issues with getting things to link correctly. Let's expand and see if it works fine now.

Reply:
Done. Lots of build failures with libgfortran_version 3, so I'll remove that